pootle/apps/pootle_statistics/migrations/0003_scorelog_translated_wordcount.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_translated_wordcount(apps, schema_editor):
+    ScoreLog = apps.get_model("pootle_statistics.ScoreLog")
+    scorelog_qs = ScoreLog.objects.select_related(
+        'submission',
+        'submission__suggestion',
+        'submission__unit'
+    )
+    for scorelog in scorelog_qs.iterator():
+        translated, reviewed = scorelog.get_paid_wordcounts()
+        ScoreLog.objects.filter(id=scorelog.id).update(
```

Comment:
As @ta2-1 pointed out in chat, using `save()` can have side effects that `update()` hasn't.